python/src/trezorlib/cli/trezorctl.py

```diff
@@ -370,6 +400,29 @@ def wait_for_emulator(obj: TrezorConnection, timeout: float) -> None:
     LOG.info(f"Waited for {end - start:.3f} seconds")
 
 
+@cli.command()
+@click.argument("directory", required=False)
+@click.option("-s", "--stop", is_flag=True, help="Stop the recording")
+@click.pass_obj
+def record(obj: TrezorConnection, directory: Union[str, None], stop: bool) -> None:
```

Comment:
this command should live under `debug`

we should also enforce that recording is only available on emulator. IIRC we can detect the emulator via its vendor string in `client.features`

Reply:
Moved under `debug` and checking the emulator in [1b9bcb0](https://github.com/trezor/trezor-firmware/pull/2548/commits/1b9bcb06c8424f0902c19ff7298f18c270d425f7)

Reply:
Check for emulator moved deeper into the "main" `record_screen` function, so it also raises when used with hardware device - [5f97686](https://github.com/trezor/trezor-firmware/pull/2548/commits/5f976867b6826ec606a8b130ee9ac614f17c828e)